helpers/award_helper.py

```diff
@@ -121,6 +121,7 @@
     (AwardType.MOST_IMPROVED_TEAM, (["most improved team"], [])),
     (AwardType.WILDCARD, (["wildcard"], [])),
     (AwardType.AUTONOMOUS, (["autonomous"], [])),
+    (AwardType.INNOVATION_CHALLENGE_SEMI_FINALIST, (["innovation", "challenge", "semi-finalist"], [])),
```

Comment:
Asking for a review because I've never done this before. Does this seem right? What's the best way to do this?

Reply:
Yep, this looks right. You can also just make it `["innovation challenge semi-finalist"]` to be extra safe, since we know the exact name of the award.